### DISCUSSION TITLE
Report generation failed due to network congestion (DFK chain)

### DISCUSSION THREAD

**Discussion by fidomullis** [pwillworth/dfkreport#21] [General]
Hi - I'm getting this error. Is this a temporary thing? 
![image](https://user-images.githubusercontent.com/113218662/199129599-9d1b1e84-1e5f-454a-b9ff-a11332f5c67f.png)

**Comment by pwillworth** [comment]
I wouldn't be surprised to see that occasionally, but are you seeing it every time you try to run a report?

**Comment by fidomullis** [comment]
Yeah, most of the time in the last few days. I've only managed to export one month's worth of transactions out of maybe 20 tries.

**Reply by pwillworth** [reply]
Wow, yeah that's more than expected, I'm not experiencing that.
I recently had to change the initial transaction list lookup to a new service which is kinda new and still has some bumps in it.
The service provider did say they plan on releasing a new version of their API, but will be a couple months.
If it continues to be bad for you and you want to dig into it more, I could try reproducing the issue using your wallet address and raise an issue with support on it if I can.

**Comment by fidomullis** [comment]
Yes, please.
It just won't generate any reports for me.

**Reply by pwillworth** [reply]
Got it, I'll do some testing.

**Reply by pwillworth** [reply]
@fidomullis I made an update to be more efficient and call that API less frequently.  I was able to run your report a few times in a row without issue.  Can you see how it's working for you now?

**Reply by fidomullis** [reply]
Thanks heaps! It seems to be working now. I've done a few test exports, no hic-ups.

**Comment by fidomullis** [comment]
Hi pwillworth,
I am again getting a similar message, but on one wallet address only. Four other wallets export fine.
![image](https://github.com/pwillworth/dfkreport/assets/113218662/48243411-6983-4865-8d04-b750b9349a17)

**Reply by pwillworth** [reply]
Hey, can you try it again now?  I think I identified an error with processing duels entries that I put a fix in for.

**Comment by fidomullis** [comment]
Yes, it's the wallet where I duel, and yes, it works now! Thanks very much!
And once again, thank you very much for the tool. I've tried a bunch and while Lila's ledger isn't perfect it's the best I've come across. Cheers!
